cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -311,6 +313,12 @@ def parse_harvard_opinions(reporter, volume, make_searchable):
             short_data = parse_extra_fields(soup, short_fields, False)
             long_data = parse_extra_fields(soup, long_fields, True)
 
+            if len(data["docket_number"]) > 5000:
```

Comment:
Did 5,000 wind up being the magic number? Can we put it in a variable in the models file, say, so it can be used later?

Reply:
This number can fluctuate based on the entry.   I dont have a good handle on what it should actually be.

Reply:
I see. It looks like the length is content-dependent because it has to do with the compressed length of the string: 

https://dba.stackexchange.com/questions/217087/why-not-error-index-row-size-xxxx-exceeds-maximum-2712-for-index-foo

Could we catch the stacktrace and do the tweak only when we truly need it?

Reply:
I just worry that 5k is too low. We're going to be hitting this at least a few times, so we should be sure to get it right.